src/main/java/com/cburch/logisim/fpga/data/DriveStrength.java

```diff
@@ -9,12 +9,11 @@
 
 package com.cburch.logisim.fpga.data;
 
-import java.util.Iterator;
 import java.util.LinkedList;
 import java.util.List;
 
 public class DriveStrength {
-  public static String GetContraintedDriveStrength(char id) {
+  public static String getContraintedDriveStrength(char id) {
```

Comment:
I am not familiar with an English word "contrainted". Is that a technical term (I am not an expert on fpgas)? Or is there a better word (such as constrained)?

Reply:
This is definitely a typo, it should be "constrained"

Reply:
I completely missed this one ^^'